menus/sgd/tools.cfg

```diff
@@ -172,3 +172,27 @@ function sbasename {
     export $1
 
 }
+
```

Comment:
Need to test this code to see who it works but overall is ok for me.

Reply:
:+1: